projects/1.20/assets/doggy-talents-next/doggytalents/lang/zh_cn.json

```
@@ -0,0 +1,824 @@
+{
+    "block.doggytalents.dog_bed": "狗窝", 
+    "block.doggytalents.dog_bed.set_owner_help": "空手时，按住潜行键+右键（或手持骨头右键点击）狗窝，最近的站立狗狗会过来并认领此床！若狗狗不幸死亡，右键点击狗窝即可复活！", 
+    "block.doggytalents.dog_bed.owner": "这是%s的床！",
+    "block.doggytalents.dog_bed.reclaim": "%s重新认领了它的床",
```

英语原文为“%s has reclaimed %s bed.“，应为：
```suggestion
    "block.doggytalents.dog_bed.reclaim": "%s重新认领了%s的床。",
```

请问，测试过没有格式规整错误吗？同时，还缺少了标点符号。